c++.html.markdown

```diff
@@ -1000,24 +1000,26 @@ cout << get<5>(concatenated_tuple) << "\n"; // prints: 'A'
 
 // Vector (Dynamic array)
 // Allow us to Define the Array or list of objects at run time
-#include<vector>
-vector<Data_Type> Vector_name; // used to initialize the vector
+#include <vector>
+typedef string my_vector_type;
```

Comment:
why?

Reply:
hi @vendethiel, thanks for you feedback. you mean the white space and the typedef?
I find it more useful and consistent when each subtopic is kind of cohesive. before this PR, there was "vector_name" and "Vector_name". The white space in the include has not really been questioned, I guess? :)

Reply:
Sorry, no. I mean why the `typedef`

Reply:
It got me confused at first because I forgot the doc has `using namespace std;` >_>

Reply:
of course we could try to have each subtopic written as a valid translatable program, here i.e. with taking care of the namespace and all includes and having a main() with a valid return, but I guess this is too fat and against the idea of learnxiny... here we could also drop the typedef because an experienced programmer will know what to do with a vector<data_type> my_vector, but I kept it to have a reader understand more quickly what this is about. at least I hoped so :)

Reply:
I don't understand why `typedef string vector_type; vector<vector_type>` is more understandable than `vector<string>`.